clients/include/blas3/testing_hemm.hpp

```diff
@@ -210,14 +211,13 @@ void testing_hemm(const Arguments& arg)
     T h_alpha = arg.get_alpha<T>();
     T h_beta  = arg.get_beta<T>();
 
-    size_t  rows = (side == HIPBLAS_SIDE_LEFT ? N : M);
-    int64_t K    = (side == HIPBLAS_SIDE_LEFT ? M : N);
+    size_t dim_A = (side == HIPBLAS_SIDE_LEFT ? M : N);
 
     hipblasLocalHandle handle(arg);
 
     // check here to prevent undefined memory allocation error
-    bool invalid_size = M < 0 || N < 0 || ldc < M || ldb < M || lda < K;
-    if(invalid_size || !M || !N)
+    bool invalid_size = M < 0 || N < 0 || ldc < M || ldb < M || lda < dim_A;
+    if(M < 0 || N < 0 || ldc < M || ldb < M || lda < dim_A)
```

Comment:
This is just the same as `if(!invalid_size)` then if we're having !M and !N pass through.

Reply:
Good catch, I will change it.